src/encoded/static/components/item-pages/components/tables/SimpleFilesTable.js

```diff
@@ -4,19 +4,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
 import { console } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
-import { ItemPageTable, ItemPageTableIndividualUrlLoader } from './ItemPageTable';
+import { ItemPageTable, ItemPageTableIndividualUrlLoader, ItemPageTableSearchLoader, ViewMoreResultsBtn } from './ItemPageTable';
 import { expFxn } from './../../../util';
 
-
+/**
+ * @todo
+ *   - Maybe make custom detail pane for type=File
+ *   - Maybe wrap in container element to match ExperimentSetTables counterpart, idk.
+ */
 export const SimpleFilesTable = React.memo(function SimpleFilesTable(props){
-    const { results, loading } = props;
+    const { results, loading, countTotalResults, hrefWithoutLimit } = props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['hrefWithoutLimit' is missing in props validation (react/prop-types)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4510119)